tools/cli.js

```diff
@@ -29,7 +29,14 @@ function complete(input, completions) {
   return completions.filter(c => c.startsWith(input));
 }
 
-function tryComplete(input, completer) {
+function tryComplete(input, completions) {
+  if (!completions) return input;
+  completions = complete(input, completions);
+  if (completions.length === 1) return completions[0];
+  return input;
+}
+
+function tryCompleter(input, completer) {
```

Comment:
@aqrln yeah, I totally agree on this one but I couldn't come up with a better name, what's your opinion on `tryToComplete` for the previous one and this one to remain `tryCompleter`?